docs/guides/examples-guide.rst

```diff
@@ -0,0 +1,207 @@
+.. _dfhack-examples-guide:
+
+DFHack Example Configuration File Guide
+=======================================
+
+The :source:`hack/examples <data/examples>` folder contains ready-to-use
+examples of various DFHack configuration files. You can use them by copying them
+to appropriate folders where DFHack and its plugins can find them (details
+below). You can use them unmodified, or you can customize them to better suit
+your preferences.
+
+The ``init/`` subfolder
+-----------------------
+
+The :source:`init/ <data/examples/init>` subfolder contains useful DFHack
+`init-files` that you can copy into your main Dwarf Fortress folder -- the same
+directory as ``dfhack.init``.
+
+:source:`onMapLoad_dreamfort.init <data/examples/init/onMapLoad_dreamfort.init>`
```

Comment:
The fact that these links don't work until the PR is merged makes me want to make `:source:` a little smarter... but that's definitely out of scope here :)